taxonomy-editor-frontend/src/pages/editentry/ListAllOtherProperties.jsx

```diff
@@ -0,0 +1,92 @@
+import { Paper, Stack, TextField, Typography } from "@mui/material";
+import { useEffect, useState } from "react";
+
+const ListAllOtherProperties = ({ props }) => {
+    const [nodeObject, setNodeObject] = useState(null);
+    useEffect(() => {
+        setNodeObject(props);
+    }, [props])
+
+    // Stores 2 letter language code (LC) of the tags
+    let languageCode = ''; 
+    // Storing keys and values that needs to be rendered for editing
+    let toBeRendered = {} 
+    // Used for conversion of LC to long-form
+    let languageNames = new Intl.DisplayNames(['en'], {type: 'language'}); 
+
+    if (nodeObject) {
+        Object.keys(nodeObject).forEach((key) => {
+            if (key.startsWith('tags') && 
+                !key.includes('ids')) {
+                    languageCode = key.slice(-2);
+                    toBeRendered[languageCode] = nodeObject[key]
+                }
+        })
+    }
+
+    // Helper function used for changing state of properties
+    function changeDataComment(key, value) {
```

Comment:
nit: if this is only used for the comment you could remove the key and just pass the value